SolrNet/Impl/SolrPostConnection.cs

```diff
@@ -24,7 +24,9 @@ public PostSolrConnection(ISolrConnection conn, string serverUrl)
 			    this.serverUrl = serverUrl;
 		    }
 
-		    public string Post(string relativeUrl, string s)
+	      public string ServerUrl => serverUrl;
```

Comment:
This should be a Property and not a Field. You can copy the implementation from `SolrConnection`. 
There you can see the we also validate the input to make sure it is an address we can use.

Reply:
Btw, I was using the new syntax for expression-bodied properties.
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/expression-bodied-members

Reply:
Sorry, you are totally right and thanks for showing me. I didn’t know the short read-only version, I only knew the one with the get/set keywords .